modules/twinkleblock.js

```diff
@@ -73,9 +73,7 @@ Twinkle.block.callback = function twinkleblockCallback() {
 				label: 'Add block template to user talk page',
 				value: 'template',
 				tooltip: 'If the blocking admin forgot to issue a block template, or you have just blocked the user without templating them, you can use this to issue the appropriate template. Check the partial block box for partial block templates.',
-				// Disallow for IP ranges
-				checked: !Morebits.ip.isRange(relevantUserName),
-				disabled: Morebits.ip.isRange(relevantUserName)
+				checked: true
```

Comment:
```suggestion
				// Disallow for (initial) IP ranges
				checked: !Morebits.ip.isRange(relevantUserName),
				disabled: Morebits.ip.isRange(relevantUserName)
```

I think actually we want to keep this.  Although not (yet) supported, when eventually we have `wgRelevantUserName` for IP ranges, this will set the default for those cases.  That is, it won't make a difference now, since this will never get triggered atm, but will be necessary to be restored when it does come.